core/src/main/kotlin/org/evomaster/core/problem/externalservice/httpws/service/HttpWsExternalServiceHandler.kt

```diff
@@ -196,7 +198,7 @@ class HttpWsExternalServiceHandler {
             val action = HostnameResolutionAction(it.remoteHostName, hostnameLocalAddressMapping[it.remoteHostName]!!)
             output.add(action)
         }
-        return output
+        return output.distinct()
```

Comment:
this likely would not work, as `HostnameResolutionAction` has no overridden `equals` based on `hostname`. but, anyway, already in `hostnameResolutionInfos` they should be kept distinct

Reply:
this is unnecessary now, since the insertion logic is fixed now.